jsapp/xlform/src/model.configs.coffee

```diff
@@ -150,6 +153,9 @@ module.exports = do ->
     date:
       label:
         value: "Enter a date"
+    range:
+      label:
+        value: "Choose a range"
```

Comment:
Let's call this "Enter a number within a specified range".
TODO: [figure out](https://www.flowdock.com/app/kobotoolbox/kobo/threads/IroY1P_E_Bfz0my8UzU9qOY_27L) where this actually appears ;)